resources/lib/svtplay.py

```diff
@@ -268,7 +268,10 @@ def __create_dir_item(self, play_item):
             return
         info = play_item.info
         fanart = play_item.fanart if play_item.item_type == PlayItem.VIDEO_ITEM else ""
-        self.__add_directory_item(play_item.title, params, play_item.thumbnail, folder, False, info, fanart)
+        title = play_item.title
+        if play_item.season_title:
```

Comment:
Add condition `play_item.item_type == PlayItem.VIDEO_ITEM` to conditions

Reply:
Updated